frontend/src/app/modules/website/edit/edit.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit } from '@angular/core';
+import { Entity } from '../../shared/components/entity-list/entity-list.component';
+import { WebsiteService } from '../website.service';
+import { ActivatedRoute, Router } from '@angular/router';
+
+@Component({
+  selector: 'app-edit',
+  templateUrl: './edit.component.html',
+  styleUrls: ['./edit.component.scss']
+})
+export class EditComponent implements OnInit {
+  website: Entity = {} as Entity;
+  editorSaveEndpoint = '';
+  isEditorMode = false;
+
+  constructor(
+    private websiteService: WebsiteService,
+    private router: Router,
+    private route: ActivatedRoute
+  ) {}
+
+  ngOnInit() {
+    this.setEditorMode();
+    this.websiteService.setActiveWebsiteId(Number(this.route.snapshot.paramMap.get('id'))??-1);
```

Comment:
this is what I am talking bout a lot of code has been repeated, move this entity list and it should call the necessary functions usingg service an dupdate entity list.

On changing to email list, ,it should fetch emails and refresh the list